docs/reference/setup/install/docker.asciidoc

```diff
@@ -121,6 +121,8 @@ docker-compose up
 
 endif::[]
 
+WARNING: This is only an example of how to use `docker-compose`. Production clusters should always be 3 or more nodes.
```

Comment:
Hmm...it's fine to start out even in a production with a single node or two (configured correctly). It's not ideal from a resiliency perspective but I don't think we should say the minimum for production is three.